src/App.js

```diff
@@ -24,19 +37,22 @@ class App extends Component {
   returnToMain = () => {
     this.setState({ onMainPage: true })
   }
+
   render() {
     return (
     this.state.onMainPage ?
       (<main>
         <h1>Rancid Tomatillos</h1>
+        { this.state.error && <p>Oops! Something went wrong. Refresh and try again.</p> }
```

Comment:
What if you held that string  as the value of `this.state.error` instead of a boolean?

Reply:
Delilah - what are your thoughts on this? Would that be something like...
{this.state.error && <p>{this.state.error}</p>}
Am I interpreting this correctly?